AnnotatoSharedLibrary/Sources/AnnotatoSharedLibrary/Models/Document.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+public class Document: Codable {
+    public var id: UUID?
+
+    public private(set) var name: String
+
+    public let ownerId: UUID
+
+    public let baseFileUrl: String
```

Comment:
```suggestion
    public var id: UUID?
    public private(set) var name: String
    public let ownerId: UUID
    public let baseFileUrl: String
```

Can we standardise to have to not have line breaks?